src/averageAges.js

```diff
@@ -37,7 +52,22 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter((person) => person.sex === 'f');
+  let sorted = women;
+
+  if (withChildren === true) {
```

Comment:
the same as in the previous function - check `withChildren` argument inside the filter's callback and if it `true` add an additional condition using `&&` operator